src/map/pc_groups.hpp

```diff
@@ -91,4 +88,36 @@ static const struct s_pcg_permission_name {
 	{ "all_permission", PC_PERM_ALLPERMISSION },
 };
 
+struct s_player_group{
+	uint32 id;
+	std::string name;
+	uint32 level;
+	bool log_commands;
+	std::unordered_map<std::string, bool> commands;
+	std::unordered_map<std::string, bool> char_commands;
+	uint32 permissions;
```

Comment:
```suggestion
	std::bitset<uint8> permissions;
```
Setting to a bitset makes this enum much smaller!

Reply:
Done!